lib/components/dropdown.vue

```diff
@@ -28,7 +28,7 @@
             <span class="sr-only">{{toggleText}}</span>
         </b-button>
 
-        <div :class="['dropdown-menu',{'dropdown-menu-right': right}]"
+        <div :class="['dropdown-menu',{'dropdown-menu-right': right, show: visible}]"
```

Comment:
Don't forget about `nav-item-dropdown` 😉 

It also has similar template and also uses the `dropdown.js` mixin